### ISSUE TITLE
tape -r babel-register equivalent for gulp-tape

### ISSUE THREAD

**Issue by matszym** [yuanqing/gulp-tape#22]
I am writing react component test. If I run it manualy with command:
`tape -r babel-register tests/counter.component.test.js`
It works just fine, but I have hard time getting it to work with gulp. Here is my task:

```
gulp.task('tape', () => {
  return gulp
  .src(paths.tests)
  .pipe(tape({
    reporter: tapSpec()
  }));
});
```

As you can see, I'm not handling the actual components files here, so adding pipe with gulp-babel wont help me. What did I miss? Can I pass somewhere `require: 'babel-register` as an option?

**Comment by pablo-peraza** [comment]
Anyone has an idea on how to do this?

**Comment by yuanqing** [comment]
This should be solved as of `0.0.10`; see https://github.com/yuanqing/gulp-tape/tree/7754fb0e2956feac10352c4d3d10fa305d2c96f1#tapeoptions
